src/site/content/en/blog/file-system-access/index.md

```diff
@@ -319,6 +319,44 @@ await directoryHandle.removeEntry('Abandoned Masterplan.txt');
 await directoryHandle.removeEntry('Old Stuff', { recursive: true });
 ```
 
+### Drag and drop integration
+
+The
+[HTML Drag and Drop interfaces](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API)
+enable web applications to accept
+[dragged and dropped files](https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/File_drag_and_drop)
+on a web page.
+During a drag and drop operation, dragged file and directory items are associated
+with file entries and directory entries respectively.
+The `DataTransferItem.getAsFileSystemHandle()` method returns a promise with a `FileSystemFileHandle` object
+if the dragged item is a file, and a promise with a `FileSystemDirectoryHandle` object if the dragged item is a directory.
+The listing below shows this in action.
```

Comment:
Note that the Drag and Drop interface's
[`DataTransferItem.kind`](https://developer.mozilla.org/en-US/docs/Web/API/DataTransferItem/kind)
will be `"file"` for both files *and* directories, whereas the File System Access API's
[`FileSystemHandle.kind`](https://wicg.github.io/file-system-access/#dom-filesystemhandle-kind)
will be `"file"` for files and `"directory"` for directories.